### PR TITLE
adding the documentation of the new SSHInfrastructureV2 variables

### PR DESCRIPTION
**Pull request by alijawadfahs**

### CODE REVIEW

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 1]
```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
 -   **javaOptions** - Java options appended to the command used to
     start the node on the remote host.
 
+-   **deploymentMode** - Specifies how the ProActive node command is started.
+The deploymentMode can take the following values:
+
+*** _autoGenerated_: when this mode is selected the command to start the
+ProActive node will be generated automatically. As a result, the ssh call to
+the nodes will attach the nodes to the infrastructure without modifications to
```
Comment (tao-xinxiu) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:141]: What does `attach the nodes to the infrastructure` mean? Does it mean starting Proactive nodes on the infrastructure?

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 2]
```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
...
+the startup command. This mode is selected by default.
+
+*** _useStartupScript_: wtarts the ProActive node using the script in the variable
```
Comment (tao-xinxiu) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: `wtarts` ? typo ?

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 3]
```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
...
+`%startupScriptStandard%`, Allowing the user to modify the startup command of the nodes.
+This mode uses the ProActive node agent identified in the `%schedulingPath%`.
+
+
+*** _useNodeJarStartupScript_: enables connecting to the SSHInfrastructureV2
+using node.jar file. This mode uses `%nodeJarUrl%` and `%startupScriptWitNodeJarDownload%`
```
Comment (tao-xinxiu) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:150]: A small suggestion: `using node.jar file` -> by launching node.jar.

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 4]
```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
...
+variables to generate the startup command.
+
++
+TIP: if deploymentMode field was set to an empty string, the autoGenerated mode will be automatically selected
++
+
+-   **nodeJarUrl** - The full URL path to download the ProActive _node.jar_
+on each node added to the hostList. The URL have to be accessible from the nodes.
```
Comment (tao-xinxiu) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:158]: `on each node` -> `on each host`
`hostList` ->  `hostsList`
`accessible from the nodes` -> `accessible from the hosts`

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 5]
```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
...
+Used only when `useNodeJarStartupScript` is selected.
+
+-   **startupScriptStandard** - Nodes startup script to launch the ProActive nodes using a ProActive node agent.
+The script by default locates java and node agent directory using `%javaPath%` and `%schedulingPath%` variables respectively.
+The user can modify or extend this script to make changes to the node before or after the startup.
+Used only when `useStartupScript` is selected.
+
+-   **startupScriptWitNodeJarDownload** - Nodes startup script to launch the ProActive nodes using _node.jar_.
+To run the ProActive nodes, this script is also expected to download and install the Java Runtime Environment,
+then download and execute ProActive _node.jar_, if they are not already provided by the node.
```
Comment (tao-xinxiu) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:169]: `already provided by the node` -> `already provided by the host`

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 6]
```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
...
++
+
+-   **nodeJarUrl** - The full URL path to download the ProActive _node.jar_
+on each node added to the hostList. The URL have to be accessible from the nodes.
+For example, `try.activeeon.com/rest/node.jar`.
+Used only when `useNodeJarStartupScript` is selected.
+
+-   **startupScriptStandard** - Nodes startup script to launch the ProActive nodes using a ProActive node agent.
+The script by default locates java and node agent directory using `%javaPath%` and `%schedulingPath%` variables respectively.
+The user can modify or extend this script to make changes to the node before or after the startup.
```
Comment (tao-xinxiu) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:164]: `to make changes to the node before or after the startup` -> `to execute commands on the host before or after the Proactive nodes startup.`

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 7]
```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
...
+Used only when `useStartupScript` is selected.
+
+-   **startupScriptWitNodeJarDownload** - Nodes startup script to launch the ProActive nodes using _node.jar_.
+To run the ProActive nodes, this script is also expected to download and install the Java Runtime Environment,
+then download and execute ProActive _node.jar_, if they are not already provided by the node.
+It uses `%nodeJarUrl%` variable to get the full URL path for downloading the _node.jar_ file.
+The user can modify or extend this script to make changes to the node before or after the startup.
```
Comment (tao-xinxiu) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:171]: `to make changes to the node before or after the startup` -> `to execute commands on the host before or after the Proactive nodes startup.`

**Review by tao-xinxiu — APPROVED** [review]
Generally LGTM. Some small suggestions. 
I think it's better to avoid confusing the term `host` (machines running nodes) and `node` (Proactive node, which could have multiple running on the same host).

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 8]
```diff
@@ -133,6 +133,50 @@ This infrastructure needs 13 arguments, described hereafter:
...
+For example, `try.activeeon.com/rest/node.jar`.
+Used only when `useNodeJarStartupScript` is selected.
+
+-   **startupScriptStandard** - Nodes startup script to launch the ProActive nodes using a ProActive node agent.
+The script by default locates Java Runtime Environment and node agent directory using `%javaPath%` and `%schedulingPath%` variables respectively.
+The user can modify or extend this script to execute commands on the host before or after the Proactive node startup.
+Used only when `useStartupScript` is selected.
+
+-   **startupScriptWitNodeJarDownload** - Nodes startup script to launch the ProActive nodes using _node.jar_.
+To run the ProActive nodes, this script is also expected to download and install the Java Runtime Environment,
```
Comment (mklkun) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:169]: To run the ProActive nodes => To run ProActive nodes

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 9]
```diff
@@ -133,6 +133,50 @@ This infrastructure needs 13 arguments, described hereafter:
...
+then download and execute ProActive _node.jar_, if they are not already provided by the host.
+It uses `%nodeJarUrl%` variable to get the full URL path for downloading the _node.jar_ file.
+The user can modify or extend this script to execute commands on the host before or after the Proactive node startup.
+Used only when `useNodeJarStartupScript` is selected.
+
++
+WARNING: If the deployment mode `autoGenerated` is selected, then the startup scripts will be disregarded.
```
Comment (mklkun) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:176]: then the startup => the startup

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 10]
```diff
@@ -133,6 +133,50 @@ This infrastructure needs 13 arguments, described hereafter:
...
+-   **deploymentMode** - Specifies how the ProActive node command is started.
+The deploymentMode can take the following values:
+
+*** _autoGenerated_: when this mode is selected the command to start the
+ProActive node will be generated automatically. As a result, the ssh call to
+the nodes will start the ProActive nodes on the infrastructure without modifications to
+the startup command. This mode is selected by default.
+
+*** _useStartupScript_: starts the ProActive node using the script in the variable
+`%startupScriptStandard%`, Allowing the user to modify the startup command of the hosts.
```
Comment (mklkun) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc:145]: , Allowing => , allowing

**Review by mklkun — COMMENTED** [review]
Some typos.

**Review by mklkun — APPROVED** [review]
LGTM

**Pull request merged**